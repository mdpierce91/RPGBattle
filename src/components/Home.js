import React from 'react';
import LoginPage from './LoginPage';

const Home = () => {
    return (
        <div className="container home-container">
            <LoginPage/>
        </div>
    );
}

export { Home as default };