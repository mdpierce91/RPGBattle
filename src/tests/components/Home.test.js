import React from 'react';
import { shallow } from 'enzyme';
import Home from '../../components/Home';

test('Should render the home page', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
});