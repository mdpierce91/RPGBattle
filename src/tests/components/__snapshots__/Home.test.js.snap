// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the home page 1`] = `
<div
  className="container home-container"
>
  <Connect(LoginPage) />
</div>
`;
